src/avram/tasks/db/verify_connection.cr

```diff
@@ -20,30 +20,28 @@ class Db::VerifyConnection < LuckyCli::Task
   end
 
   def call
```

Comment:
Was this a tabs/spaces change?

Reply:
Maybe it was tabulated with tabs and this changed automatically when redundant `begin` block has been removed.